src/Data/Nat.agda

```diff
@@ -27,6 +29,10 @@ open import Data.Nat.Properties public
   ; _≤″?_; _<″?_; _≥″?_; _>″?_
   )
 
+infix 4 _≢0
+_≢0 : ℕ → Set
```

Comment:
Does adding this function really belong in this PR ?  Or did it slip in by accident?

Reply:
I moved it from `Data.Rational.Unnormalised` because I spotted the
code duplication with `Data.Rational.Base` during the refactoring.
Removed from now.